events/importer/turku_old_events.py

```diff
@@ -0,0 +1,794 @@
+            json_event = json_mother_event['event']
+
+            if json_event['drupal_nid']:
+                for x in childList:
+                    for k, v in x.items():
+                        if json_event['drupal_nid'] == k:
+                            try:
+                                child = Event.objects.get(origin_id=k)
+                                mother = Event.objects.get(origin_id=v)
+                                try:
+                                    Event.objects.update_or_create(
+                                        id = child.id,
+                                        defaults = {
+                                        'date_published' : mother.date_published,
+                                        'provider' : mother.provider,
+                                        'provider_fi' : mother.provider_fi,
+                                        'provider_sv' : mother.provider_sv,
+                                        'provider_en' : mother.provider_en,
+                                        'description' : mother.description,
+                                        'description_fi' : mother.description_fi,
+                                        'description_sv' : mother.description_sv,
+                                        'description_en' : mother.description_en,
+                                        'short_description' : mother.short_description,
+                                        'short_description_fi' : mother.short_description_fi,
+                                        'short_description_sv' : mother.short_description_sv,
+                                        'short_description_en' : mother.short_description_en,
+                                        'location_id' : mother.location_id,
+                                        'location_extra_info' : mother.location_extra_info,
+                                        'location_extra_info_fi' : mother.location_extra_info_fi,
+                                        'location_extra_info_sv' : mother.location_extra_info_sv,
+                                        'location_extra_info_en' : mother.location_extra_info_en,
+                                        'info_url' : mother.info_url,
+                                        'info_url_fi' : mother.info_url_fi,
+                                        'info_url_sv' : mother.info_url_fi,
+                                        'info_url_en' : mother.info_url_fi,
+                                        'super_event' : mother}
+                                        )
+                                except Exception as ex: print(ex)
+                            except Exception as ex: pass
+
+                            try:
+                                # -> Get object from Event.
+                                child = Event.objects.get(origin_id=k)
+                                mother = Event.objects.get(origin_id=v)
```

Comment:
Unnecessary get, already fetched above.

Reply:
I will keep the child get since child values got updated with update_or_create